app/helpers/commonwealth_vlr_engine/images_helper_behavior.rb

```diff
@@ -12,6 +12,8 @@ def collection_gallery_url(document, size)
       if exemplary_image_pid
         if document[blacklight_config.hosting_status_field.to_sym] == 'harvested' || document['exemplary_image_iiif_bsi'] == false
           filestream_disseminator_url(document[:exemplary_image_key_base_ss], 'image_thumbnail_300')
+        elsif document[:destination_site_ssim]&.include?('newspapers')
+          iiif_image_url(exemplary_image_pid, { region: 'pct:4,3,90,67', size: "#{size},#{size}" })
```

Comment:
is it wise to specify a hardcoded region here? maybe set a const to store the string instaed of setting the string here ie

`IIIF_DEFAULT_REGION='pct:4,3,90,67'`